gapic-generator/lib/google/gapic/generators/base_generator.rb

```diff
@@ -15,7 +15,7 @@
 # limitations under the License.
 
 require "action_controller"
-require "protobuf/descriptors"
+require "google/protobuf/compiler/plugin.pb"
```

Comment:
It might be wise to replace the `require "protobuf/descriptors"` statements in the cloud gem as well.

Reply:
There are none in the cloud gem.